lib/kernel-manager.js

```diff
@@ -31,13 +31,15 @@ class KernelManager {
     this._kernelSpecs = this.getKernelSpecsFromSettings();
   }
 
-  startKernelFor(grammar: atom$Grammar, onStarted: (kernel: Kernel) => void) {
+  startKernelFor(
+    grammar: atom$Grammar,
+    editor: ?atom$TextEditor,
```

Comment:
We now need to change the function call [here](https://github.com/nteract/hydrogen/blob/master/lib/main.js#L305) and [here](https://github.com/nteract/hydrogen/blob/master/lib/main.js#L435)